src/gui/FileBrowser.cpp

```diff
@@ -653,6 +685,14 @@ void FileBrowserTreeWidget::contextMenuEvent(QContextMenuEvent* e)
 		contextMenu.addAction(QIcon(embed::getIconPixmap("folder")), tr("Show in %1").arg(fileManager),
 			[=] { FileRevealer::reveal(file->fullName()); });
 
+
+		contextMenu.addAction(
+		   !_isFavorite ? tr("Add favorite file") : tr("Remove favorite file"),
+		std::function([=, this] {
+				!_isFavorite ? addFavorite(file->fullName()) : removeFavorite(file->fullName());
+			})
+		);
```

Comment:
Is this properly formatted? I also try not to capture `=` in lambdas since it usually copies more than what is necessary in the scope of the lambda. [Capture only what you really need or capture by reference is the lambda is only used locally](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f52-prefer-capturing-by-reference-in-lambdas-that-will-be-used-locally-including-passed-to-algorithms).

Also, why is this duplicated for both directories and files? The action names are the only thing that's different. `Add favorite` and `Remove favorite` works. No need to be specific about if its a file or folder.